core/sql/optimizer/NAFileSet.h

```diff
@@ -72,6 +72,13 @@ enum FileOrganizationEnum
   KEY_SEQUENCED_FILE,
   HASH_FILE
 };
+
+enum RowFormatEnum {
+  SQLMX_UNKNOWN_FORMAT,
+  SQLMX_ROW_FORMAT,
+  SQLMX_ALIGNED_ROW_FORMAT,
+  SQLMX_HBASE_FORMAT
```

Comment:
SQLMX_ALIGNED_ROW_FORMAT is already used in NATable.h for a flag. Should we avoid this conflict (renaming either one). Also, I'm wondering whether this should go into ComSmallDefs, since it is fairly fundamental.